src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4

```diff
@@ -101,7 +101,7 @@ DECIMAL_VALUE
     : DECIMAL_DIGITS
     ;
 
-BOOLEAN_VALUE
+booleanValue
```

Comment:
Need to change to this so that Antlr dont treat it as terminate node.

Reply:
What does this mean? Why is it treated as terminate node?

Reply:
antlr treats `CAP_CASE` as terminate node, that it can not differentiate `TRUE | FALSE` anymore.  So that it does not take in `foo=True` as parameter.